Reemit.Decompiler.Clr/Disassembler/OpcodeInfoComparer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace Reemit.Decompiler.Clr.Disassembler;
+
+public class OpcodeInfoComparer : IEqualityComparer<OpcodeInfo>
+{
+    public static OpcodeInfoComparer Instance = new OpcodeInfoComparer();
```

Comment:
Probably should make this one `readonly`.

Reply:
Done